.ruby-version

```diff
@@ -1 +1 @@
-2.5.3
+2.6.3
```

Comment:
It looks like the pull request is trying to update the Ruby version from 2.5 to 2.6.  We might want to avoid merging this file.  Thoughts?

Reply:
I'm okay with leaving this file unchanged.